File: Assiment/Assignment7/Q8.c
#include <stdio.h>
//Q8. Merge two arrays

void main () {
    int arr[3]= {1,2,3};
    int brr[3] = {4,5,6};
    int crr [6];
    int i=0;
    


       for (int j = 0; j < 3; j++){
                
                 crr[i] = arr[j];
                 i++;
                }

            
                for (int j = 0; j <3;j++){
                           crr[i] = brr[j];
                     i++;
                }
               printf("marge array");     
       for (int i = 0; i < 6; i++){
         printf("%d",crr[i]);
       }
    
    }

File: Core Java/Core_java_Revision/17th sept/Array/Q8.java
//8.Merge two arrays


class test{
           public static void main(String [] args){
           int arr [] = {1,2, 3, 4, 5};
           int brr [] = {6,7,8,9,10};
           int crr [] =  new  int [arr.length + brr.length];

          for(int i = 0; i < arr.length;i++)
         {
                 crr[i] = arr[i];    
         }
          
         for(int  i = 0; i < brr.length;i++){
               crr[arr.length + i] = brr[i];
         }
         
         
        for(int i = 0 ; i < crr.length; i++){

         System.out.println(crr[i]);}  

}}